Multiplicity.Packets/NPCUpdate.cs

```diff
@@ -32,23 +30,16 @@ public class NPCUpdate : TerrariaPacket
 
         public float VelocityY { get; set; }
 
-        public byte Target { get; set; }
-
-        public NPCUpdateFlags Flags { get; set; }
```

Comment:
scratch that, I actually think you should bring them back. (see my comment on the player controls stuff)